### ISSUE TITLE
Guide?

### ISSUE THREAD

**Issue by KoolenDasheppi** [cretz/owncast#3]
I figured out how to compile it, just not sure on how exactly to use it. A README.md would do some good for this repo.

**Comment by cretz** [comment]
Yeah, it's just toy PoC code and I probably won't get back to it until https://bugs.chromium.org/p/chromium/issues/detail?id=809249 is marked resolved.

**Comment by askz** [comment]
its closed ! https://bugs.chromium.org/p/chromium/issues/detail?id=809249

**Comment by cretz** [comment]
I am starting to get an inkling of revisiting this now :-) Basically this is just a poc that had tools to replace the public key cert inside the Chromium binary and use mdns et al to make Chrome think it was a Chromecast. I suspect now that it's further along, I can have it receive non-DRM'd video fairly easily. But with all I have going on, it may be a while before I revisit.
